corehq/apps/domain/templates/domain/pro_bono/domain.html

```diff
@@ -1,4 +1,9 @@
 {% extends "style/base_section.html" %}
+{% load hq_shared_tags %}
+
+{% block js %}{{ block.super }}
+    <script src="{% static 'domain/js/pro-bono.js' %}"></script>
+{% endblock %}
```

Comment:
It doesn't work because the pro bono templates don't extend pro_bono/base.html, they include it. Perhaps I should rename `base.html`, since we virtually always extend templates named `base`.